images/metering-ansible-operator/roles/meteringconfig/defaults/main.yml

```diff
@@ -363,6 +366,17 @@ _ocp_enabled_overrides:
         prometheus:
           url: "https://prometheus-k8s.openshift-monitoring.svc:9091/"
 
+  presto:
+    spec:
+      config:
+        connectors:
+          prometheus:
+            enabled: true
+            uri: "https://prometheus-k8s.openshift-monitoring.svc:9091/"
+            bearer:
+              tokenFile: "/var/run/secrets/kubernetes.io/serviceaccount/token"
+            useServiceAccountToken: true
```

Comment:
```yaml
auth:
  bearerTokenFile: "/var/run/..."
  useServiceAccountToken: true
```

I'm not sure we need to specify the `useServiceAccountToken` either as the default value is set to `true`.

Reply:
in `values.yaml` right? If so, then yes, I agree we don't need to set it here.

Reply:
Yes, in `values.yaml`

Reply:
Done in last several commits.